test/kwc-feature-list_test.html

```diff
@@ -0,0 +1,89 @@
+
+        <title>kwc-feature-list test</title>
+
+        <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+        <script src="../../web-component-tester/browser.js"></script>
+
+        <link rel="import" href="../kwc-feature-list.html">
+    </head>
+    <body>
+        <test-fixture id="basic">
+            <template>
+                <kwc-feature-list></kwc-feature-list>
+            </template>
+        </test-fixture>
+
+        <script>
+            suite('kwc-feature-list', function() {
+                var element;
+
+                setup(function () {
+                    element = fixture('basic');
+                });
+
+                test('instantiating the element works', function() {
+                    assert.equal(element.is, 'kwc-feature-list');
+                });
+
+                test('the title is undefined by default', function() {
+                    assert.equal(element.title, undefined);
+                });
+
+                test('the title can be set imperatively', function(done) {
+                    element.title = 'Features'
+                    assert.equal(element.title, 'Features');
+                    flush(function() {
+                        var title =  Polymer.dom(element.root).querySelectorAll('h2')[0];
+                        assert.equal(title.textContent, 'Features');
+                        done();
+                    });
+                });
+
+                test('the features are an empty array by default', function() {
+                    assert.isArray(element.features);
+                });
+
+                test('the features can be set imperatively', function(done) {
+                    var imageUrl = 'https://placekitten.com/300/150';
+                    element.features = [
+                        {
+                            imgSrc: imageUrl,
+                            title: "Title 1",
+                            body: "Body 1"
+                        }, {
+                            imgSrc: imageUrl,
+                            title: "Title 2",
+                            body: "Body 2"
+                        }, {
+                            imgSrc: imageUrl,
+                            title: "Title 3",
+                            body: "Body 3"
+                        }
+                    ];
+                    assert.lengthOf(element.features, 3);
+                    assert.isObject(element.features[0]);
+                    assert.equal(element.features[0].imgSrc, imageUrl);
+                    assert.equal(element.features[0].title, 'Title 1');
+                    assert.equal(element.features[0].body, 'Body 1');
+                    flush(function() {
+                        var features =  Polymer.dom(element.root).querySelectorAll('.features .feature'),
+                            image = features[0].querySelectorAll('img')
```

Comment:
I think the method you want to use here is `querySelector` instead of `querySelectorAll`.